index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <header>
+        <div class='head'> 
+            <div class="name">Yohannes Zemed </div>
+           <img src="icons/menu.png" alt='menu'/>
+        </div>
+    </header>
+    <section class="heading">
+
+         
+      <div class="main-text">
+        <div class ='text'>
+         <div>Hey there.</div>
+         <div >I'm Yohannes</div>
+        </div>
+        <div class ='text1'>I'am a software Developer</div>
```

Comment:
- [ ] [REQUIRED] On the subtitle with ''I'm a software developer'', I think you should use a `p` tag.